real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/config.java

```diff
@@ -38,13 +38,19 @@
 	public static boolean AUTONOMUS_DEFAULT = false;
 
 	// sensor default settings (true for on false for off)
-
 	public static boolean DRIVE_DEFAULT = true;
 	public static boolean IMU_DEFAULT = true;
 	public static boolean GPS_DEFAULT = true;
 	public static boolean ENCODER_DEFAULT = true;
 	public static boolean VISION_SYSTEM_DEFAULT = true;
 	
+	//sensor com ports
+	public static String COM_PORT_IMU = "COM6";
+	public static String COM_PORT_GPS_INTEGRATED = "COM5";
+	public static String COM_PORT_GPS_STANDALONE = "COM8";
+	public static String COM_PORT_ENCODER = "COM3";
+	public static String COM_PORT_DRIVE_CONTROL = "COM9";
+	
```

Comment:
maybe it's worth having a note that these lines should probably be reverted before new commits, so as not to ruin other people's days?